google/cloud/security/common/data_access/bigquery_dao.py

```diff
@@ -0,0 +1,75 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Provides the data access object (DAO) for Big Query."""
+
+from MySQLdb import DataError
+from MySQLdb import IntegrityError
+from MySQLdb import InternalError
+from MySQLdb import NotSupportedError
+from MySQLdb import OperationalError
+from MySQLdb import ProgrammingError
+
+from google.cloud.security.common.data_access import errors
+from google.cloud.security.common.data_access import project_dao
+from google.cloud.security.common.data_access.sql_queries import select_data
+# pylint: disable=line-too-long
+from google.cloud.security.common.gcp_type import bigquery_access_controls as bq_acls
+# pylint: enable=line-too-long
+from google.cloud.security.common.util import log_util
+
+LOGGER = log_util.get_logger(__name__)
+
+
+class BigqueryDao(project_dao.ProjectDao):
+    """Data access object (DAO) for Organizations."""
+
+    def get_bigquery_acls(self, resource_name, timestamp):
+        """Select the Big Query acls from a Big Query acls snapshot table.
+
+        Args:
+            resource_name (str): String of the resource name.
+            timestamp (str): String of timestamp, formatted as
+            YYYYMMDDTHHMMSSZ.
+
+        Returns:
+            list: List of Big Query acls.
```

Comment:
It looks like you actually return a dictionary keyed by the count of acls and then the acl?